modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -57,7 +57,14 @@ Runtime Fabric enables you to specify the severity level of messages that are wr
 Log levels are specified per Mule application or API proxy during deployment.
 
 [WARNING]
-Because log levels are specified when deploying a Mule application or an API proxy, you cannot change the log levels after deployment.
+
+* Because log levels are specified when deploying a Mule application or an API proxy, you cannot change the log levels after deployment.
```

Comment:
I don't think we need this line. You can set properties on new deployments, or edit existing deployments and add/remove/change properties

Reply:
Removed - thank you!